src/lib/input/autofill.ts

```diff
@@ -78,8 +80,10 @@ export class AutofillMonitor implements OnDestroy {
       }
     };
 
-    element.addEventListener('animationstart', listener, listenerOptions);
-    element.classList.add('mat-input-autofill-monitored');
+    this._ngZone.runOutsideAngular(() => {
+      element.addEventListener('animationstart', listener, listenerOptions);
```

Comment:
Can you put it back in the zone when we emit on the observable? That way users don't have to do it themselves

Reply:
Done.